### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by enis-memic**
In this following branch:

- Added 2 more sections to the page
- Added styling to both of those sections
- Fixed some errors in the README.md file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -96,3 +97,199 @@ li {
...
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-left: -30px;
+}
+
+.sidebar {
+  margin-top: 5px;
+}
```
Comment (emmiiorji) [inline style.css:275]: Your design looks great. Kindly adjust the arrows at the languages section to match the design. Consider the margin and alignment as shown in the illustration below:

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @enis-memic, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comment under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 </div>
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 My Portfolio <a name="about-project"></a>
```
Comment (GedeonTS) [inline README.md:9]: - [OPTIONAL ] Kindly make sure your README file is professional, considering all the fields are compulsory apart from the FAQ :recycle:

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @enis-memic ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of Github flow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**